tests/test_helpers/cassettes/test_assistants/test_AIAssistant_pydantic_structured_output.yaml

```diff
@@ -49,64 +52,75 @@ interactions:
       - application/json
       Date: Sun, 09 Jun 2024 23:39:08 GMT
       Server: DUMMY
+      Strict-Transport-Security:
+      - max-age=31536000; includeSubDomains; preload
       Transfer-Encoding:
       - chunked
       X-Content-Type-Options:
       - nosniff
       access-control-expose-headers:
       - X-Request-ID
-    http_version: HTTP/1.1
-    status_code: 200
+      cf-cache-status:
+      - DYNAMIC
+      openai-project:
+      - proj_7m8RjEIHmauJaSe2iK7Z48fc
```

Comment:
proj is leaking here, you can manually overwrite this (directly in the cassette)